bases/base58.js

```diff
@@ -1,7 +1,16 @@
 'use strict'
 const baseX = require('base-x')
+const bytes = require('../bytes')
+
+const wrap = obj => ({
+  encode: b => obj.encode(Buffer.from(b)),
```

Comment:
base-x depends on `Buffer` still? also you may want to `const { Buffer } = require('buffer')` in this file if we're still tied to it

Reply:
Ya, `base-x` does an `isBuffer()` check and fails with Uint8Array.

I’m a little torn on importing `Buffer` here. It feels like an upgrade that might actually put more code in the bundle. We really just need to get rid of `base-x` if we can.